clj-refactor.el

```diff
@@ -2211,6 +2217,7 @@ form."
   (-when-let (aliases (and cljr-magic-requires
                            (not (cider-in-comment-p))
                            (not (cider-in-string-p))
+                           (not (cljr--in-keyword-p))
```

Comment:
that is true. however, most likely the code in current master does the wrong thing too (have not tested yet) as it does not strip `::` from the prefix so it will never find it in the candidate list. this should be fixed too.

so in summary: